src/QUMainWindow.cpp

```diff
@@ -1027,20 +1029,18 @@ void QUMainWindow::checkForUpdate(bool silent) {
 		return;
 	}
 
-	tmp.write(reply->readAll());
-	tmp.seek(0);
```

Comment:
I removed the temporary file, not sure why this was used.

Reply:
Not sure either, but was probably not necessary.